src/components/Sum.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({ a = 2, b = 1 }) => (
```

Comment:
If you indicate that all of your props are required, default values are redundant